Gems/ROS2/Code/Source/RobotImporter/xacro/XacroUtils.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "XacroUtils.h"
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/XML/rapidxml.h>
+#include <QProcess>
+#include <QString>
```

Comment:
There is actually no need to use Qt here.

The AzFramework library has a [ProcessWatcher](https://github.com/o3de/o3de/blob/development/Code/Framework/AzFramework/AzFramework/Process/ProcessWatcher.h#L69-L70) class that can launch processes across all the support platforms